app/components/application_record_card/view.rb

```diff
@@ -36,13 +36,18 @@ def route
     end
 
     def updated_at
-      date_stamp = record.timeline&.first&.date.presence || record.created_at
-      date_text = tag.span(date_stamp.strftime("%-d %B %Y"))
+      date_text = tag.span(last_updated_date.strftime("%-d %B %Y"))
       class_list = "govuk-caption-m govuk-!-font-size-16 govuk-!-margin-top-2 govuk-!-margin-bottom-0 application-record-card__submitted"
 
       sanitize(tag.p(date_text.prepend("Updated: "), class: class_list))
     end
 
+    def last_updated_date
+      return record.updated_at if record.draft?
+
+      record.timeline&.first&.date.presence || record.updated_at
```

Comment:
do we use `&.` here to handle nil values?

Reply:
yep! i haven't changed this part of the code from the old line 39 above - but it's guarding against a case where a trainee doesn't have a timeline.

i can't actually think of an 'everyday' reason why they wouldn't have one, but i would rather keep the code working as it has been for now!